src/sources/soundsourceffmpeg.cpp

```diff
@@ -495,317 +487,47 @@ long SoundSourceFFmpeg::seek(long filepos) {
             }
         }
 
-        if (filepos == 0) {
-            readFramesToCache((SOUNDSOURCEFFMPEG_CACHESIZE - 50), -1);
+        if (frameIndex == 0) {
+            readFramesToCache((AUDIOSOURCEFFMPEG_CACHESIZE - 50), -1);
         } else {
-            readFramesToCache((SOUNDSOURCEFFMPEG_CACHESIZE / 2), filepos);
+            readFramesToCache((AUDIOSOURCEFFMPEG_CACHESIZE / 2), frameIndex);
         }
     }
 
 
-    if (m_lCacheEndByte <= (unsigned long) filepos) {
-        readFramesToCache(100, filepos);
+    if (m_lCacheEndByte <= (unsigned long) frameIndex) {
+        readFramesToCache(100, frameIndex);
     }
 
-    m_iCurrentMixxTs = filepos;
+    m_iCurrentMixxTs = frameIndex;
```

Comment:
Its 32 or 64 bit long, depending on the target architecture. 
It should allow the compiler to use SSE vectorization.